controllers/propagator/replication.go

```diff
@@ -50,7 +50,7 @@ func (r *ReplicatedPolicyReconciler) buildReplicatedPolicy(ctx context.Context,
 	root.ObjectMeta.DeepCopyInto(&replicated.ObjectMeta)
 	root.Spec.DeepCopyInto(&replicated.Spec)
 	replicated.SetName(replicatedName)
-	replicated.SetNamespace(decision.ClusterNamespace)
+	replicated.SetNamespace(clusterName)
```

Comment:
Can I in theory use a different namespace than the cluster name with hosted mode?

Reply:
I don't think so? Hosted mode still has the concept of ManagedClusters that have the same name and namespace. The difference is that in hosted mode, the config-policy-controller is pointing to a different cluster, and the addon-controller deploys the framework to a custom namespace. The hub cluster (i.e. the propagator) isn't all that affected by hosted mode.